src/js/internal/process/next_tick.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
+
+var nextTickQueue = [];
+
+module.exports.nextTick = function nextTick(callback) {
+  var args;
+  switch (arguments.length) {
+    case 1: break;
+    case 2: args = [arguments[1]]; break;
```

Comment:
We don't need the array actually.

Reply:
With the following suggestion:

```suggestion
    case 2: args = arguments[1]; break;
```